HotTips/TipOfTheDay.cs

```diff
@@ -51,8 +52,8 @@ public static void ShowWindow()
                 // Now show the dialog
                 tipOfTheDayWindow.Show();
 
-                // Mark tip as seen
-                _tipHistoryManager.MarkTipAsSeen(nextTip.globalTipId);
+                // Mark tip as seen and default like status is normal
+                _tipHistoryManager.MarkTipAsSeen(new TipHistoryInfo(nextTip.globalTipId+":1"));
```

Comment:
Appending ":1" in this parameter is not very clean. Would be better to pass the Enum into the constructor.